Helper/PaymentMethods.php

```diff
@@ -316,7 +318,7 @@ protected function getCurrentCountryCode($store, $country = null)
             return $country;
         }
 
-        if ($country = $this->getQuote()->getBillingAddress()->getCountry()) {
+        if (!is_null($isCustomerLoggedIn) && $country = $this->getQuote()->getBillingAddress()->getCountry()) {
```

Comment:
`$quote->getCustomerId()` returns an int if customer is logged in and `null` if not. Therefore I am checking whether the customer ID is not null here